### DISCUSSION TITLE
Warning text in docker-compose docs doesn't match replace directive

### DISCUSSION THREAD

**Discussion by stubents** [testcontainers/testcontainers-go#1231] [General]
Hi everyone,

I'm a little confused about the replace directive needed for the docker-compose module.
The text in the docs (https://golang.testcontainers.org/features/docker_compose/) states that a replace directive is needed until docker 22.06 is out.
As this is a rather old version now, the replace direct has been removed both in the code and in the docs. (https://github.com/testcontainers/testcontainers-go/commit/8d2cb1acfe7a0fdf8e98911e109f9ff9fdaec8dc#diff-d955f04950eb12c592b4e791abbd7288c7a429be971c632b7ed707bd54868bc0L24-L27)

However, there are still some replacements for cucumber and k8s.
So I wonder if this warning could be removed altogether or if I still need to include this and the warning message is just outdated.

**Comment by mdelapenya** [comment]
Thanks for pointing out to this @stubents. Indeed, the docker folks updated the replacements in the compose code (https://github.com/docker/compose/blob/v2/go.mod#L175-L189) and we did not update them in our compose module.

Given you have already done the research, would you like to contribute the update in our docs, or even in the compose module? 🙏

**Reply by stubents** [reply]
I can give it a try :)

**Comment by mdelapenya** [comment]
Given the fix has been applied, I'm closing this discussion. Thanks for your time here!
